pkg/cosign/bundle/protobundle_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"testing"
+
+	"github.com/sigstore/rekor/pkg/generated/models"
+)
+
+func TestMakeProtobufBundle(t *testing.T) {
+	testCases := []struct {
+		name           string
+		hint           string
+		rawCert        []byte
+		rekorEntry     *models.LogEntryAnon
+		timestampBytes []byte
+	}{
+		{
+			name:           "hint with timestamp",
+			hint:           "asdf",
+			rawCert:        []byte{},
+			rekorEntry:     nil,
+			timestampBytes: []byte("timestamp"),
+		},
+		{
+			name:           "only cert",
+			hint:           "",
+			rawCert:        []byte("cert stuff"),
+			rekorEntry:     nil,
+			timestampBytes: []byte{},
+		},
```

Comment:
nit, can we add a test case for rekor?

Reply:
Not easily, although maybe I'm thinking about this the wrong way.

I spent some time trying to mock out a hashedrekord / intoto entry, but the verification that `tle.GenerateTransparencyLogEntry()` performs is pretty substantial. Otherwise I could try to take a production entry and work backwards to populate a `LogEntryAnon`? What do you think?

Reply:
I was able to co-opt / adapt a test from https://github.com/sigstore/rekor/blob/main/pkg/tle/tle_test.go